src/transactions/types.ts

```diff
@@ -219,6 +228,25 @@ export interface TokenApproval {
   status: TransactionStatus
 }
 
+export interface DepositOrWithdraw {
+  networkId: NetworkId
+  type: TokenTransactionTypeV2.Deposit | TokenTransactionTypeV2.Withdraw
+  transactionHash: string
+  timestamp: number
+  block: string
+  fees: Fee[]
+  providerName: string | undefined
```

Comment:
I was thinking we could also keep `providerId` and would use it first.
`providerName` would be used as a fallback. For all the other dapps Zerion already knows about.